app/models/game_purchase.rb

```diff
@@ -47,6 +47,12 @@ class GamePurchase < ApplicationRecord
       allow_nil: true
     }
 
+  validates :replay_count,
+    numericality: {
```

Comment:
also this should only allow integers

Reply:
["You're therefore correct in saying that it's not required for native Integer values from the DB."](https://stackoverflow.com/questions/33324699/rails-4-validations-numericality-only-integer-true-necessary-on-intege)